website/docs/d/en_smtp_user.html.markdown

```diff
@@ -0,0 +1,54 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_en_smtp_user"
+description: |-
+  Get information about en_smtp_user
+subcategory: "Event Notifications API"
+---
+
+# ibm_en_smtp_user
+
+Provides a read-only data source to retrieve information about an en_smtp_user. You can then reference the fields of the data source in other resources within the same configuration by using interpolation syntax.
+
+## Example Usage
+
+```hcl
+data "ibm_en_smtp_user" "en_smtp_user" {
+	en_smtp_user_id = "en_smtp_user_id"
+	instance_id = ibm_en_smtp_user.en_smtp_user_instance.instance_id
+	user_id = ibm_en_smtp_user.en_smtp_user_instance.user_id
+}
+```
+
+## Argument Reference
+
+You can specify the following arguments for this data source.
+
+* `en_smtp_user_id` - (Required, Forces new resource, String) Unique identifier for SMTP.
```

Comment:
In schema we see 
<img width="1147" alt="Screenshot 2024-05-08 at 2 11 48 PM" src="https://github.com/IBM-Cloud/terraform-provider-ibm/assets/24930306/a08a274f-7782-42fd-94dd-2124a0d84d83">
`en_smtp_user_id` we don't have this field defined in schema. doc(example, argument and testcases are referring it )